tools/paas_dashboard/dashboards/paas-overview.erb

```diff
@@ -0,0 +1,19 @@
+<script type='text/javascript'>
+$(function() {
+  Dashing.numColumns = 3
+  Dashing.widget_base_dimensions = [600, 500]
```

Comment:
Sure, but that's JS implementation. We could have done that with CSS perhaps, which would be a top up to the JS Smashing uses?